File: src/main/java/yaasl/server/persistence/PilotRoleRepository.java
package yaasl.server.persistence;

import org.springframework.data.repository.CrudRepository;
import yaasl.server.model.Pilot;
import yaasl.server.model.PilotRole;

public interface PilotRoleRepository extends CrudRepository<PilotRole, Long> {

    PilotRole findByDescription(String description);

}

File: src/main/kotlin/yaasl/server/persistence/PilotRoleRepository.kt
package yaasl.server.persistence

import org.springframework.data.repository.CrudRepository
import yaasl.server.model.Pilot
import yaasl.server.model.PilotRole

interface PilotRoleRepository : CrudRepository<PilotRole, Long> {

    fun findByDescription(description: String): PilotRole

}
